git_machete/client.py

```diff
@@ -2155,7 +2155,7 @@ def __derive_remote_and_github_org_and_repo(self,
             f'Multiple non-origin remotes correspond to GitHub in this repository: '
             f'{", ".join(remote_and_github_org_and_repo.keys())} -> aborting. \n'
             f'You can also select the repository by providing 3 git config keys: '
-            f'`machete.github.[remote,organization,repository]`\n')
+            f'`machete.github.{{remote,organization,repository}}`\n')
```

Comment:
Or just drop `f` and add `noqa`!

Reply:
`noqa` should only be used where it is not at all possible without it.
If the f-string style is adopted, it should be used. Using {{ to denote { in f-string is the f-string-style